airbyte-integrations/connector-templates/source-configuration-based/README.md.hbs

```diff
@@ -48,10 +48,14 @@ docker run --rm -v $(pwd)/secrets:/secrets -v $(pwd)/integration_tests:/integrat
 ## Testing
 
 #### Acceptance Tests
-Customize `acceptance-test-config.yml` file to configure tests. See [Source Acceptance Tests](https://docs.airbyte.io/connector-development/testing-connectors/source-acceptance-tests-reference) for more information.
+Customize `acceptance-test-config.yml` file to configure tests. See [Source Acceptance Tests](https://docs.airbyte.com/connector-development/testing-connectors/source-acceptance-tests-reference) for more information.
 If your connector requires to create or destroy resources for use during acceptance tests create fixtures for it and place them inside integration_tests/acceptance.py.
 
-To run your integration tests with docker
+To run your integration tests with docker:
+```
+docker build . -t airbyte/source-exchange-rates-tutorial:dev
+python -m pytest integration_tests -p integration_tests.acceptance
```

Comment:
This runs integration tests using python - this should suggest using acceptance-tests-docker.sh

Reply:
@sherifnada ah, thanks for that! Updated the command. 👍